Cookbook/Style-guide/README.md

```diff
@@ -958,6 +959,28 @@ In larger expressions, optional parentheses can sometimes make code read more cl
 let playerMark = (player == current ? "X" : "O")
 ```
 
+## Commas
+
+Trailing commas in arrays and dictionaries should be avoided.
```

Comment:
I fundamentally disagree with this.

Reply:
Thanks for your opinion. 👍 :trollface:

Reply:
I double @mluisbrown on that

Reply:
I love trailing comas, I forget the comas every time I add new stuff and off course it fails to build